pages/makes.jsx

```diff
@@ -120,6 +121,7 @@ var MePage = React.createClass({
       });
       pageContent = (
         <div>
+          <p className="context">In the fall of 2015, we retired Popcorn Maker and Appmaker, as well as older versions of Thimble and X-Ray Goggles. Any makes you created with these tools are still accessible below. Projects created with the new X-Ray Goggles, Thimble, or Webmaker are accessible through those respective platforms.</p>
```

Comment:
i'm going to replace "makes" with "projects" and push this live today.  we can work on tweaks later so they dont act like blocker.  :+1: :smile: